src/background/engineManager.ts

```diff
@@ -544,6 +553,45 @@ export class EngineManager {
     }
     return "ok";
   }
+
+  private lock = new AsyncLock({
+    timeout: 1000,
+  });
+  private lockKey = "write";
+
+  /**
+   * サードパーティ向けの設定ファイルを書き出す
+   */
+  async writeEngineInfoFor3rdParty() {
```

Comment:
めっちゃ細かいですが、sつけて全情報書き出せる感じを匂わせられれば！

```suggestion
  async writeEngineInfosFor3rdParty() {
```